simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/linked/LinkedPolicy.java

```diff
@@ -40,12 +44,13 @@
  *
  * @author ben.manes@gmail.com (Ben Manes)
  */
-public final class LinkedPolicy implements KeyOnlyPolicy {
+public final class LinkedPolicy implements Policy {
   final Long2ObjectMap<Node> data;
   final PolicyStats policyStats;
   final EvictionPolicy policy;
   final Admittor admittor;
   final int maximumSize;
+  int currentSize;
```

Comment:
nit: please put `currentSize` after the final fields
explanation: A nice style in Java is to declare fields as,
```
{statics}
<break>
{final instance fields}
<break>
{mutable instance fields}
```